openquake/server/views.py

```diff
@@ -517,6 +517,48 @@ def calc_remove(request, calc_id):
                             content_type='text/plain', status=500)
 
 
+def job_share(user_level, calc_id, revert=False):
+    if user_level < 2:
+        return HttpResponseForbidden()
+    try:
+        message = logs.dbcmd('share_job', calc_id, revert)
+    except dbapi.NotFound:
+        return HttpResponseNotFound()
+
+    if 'success' in message:
+        return HttpResponse(content=json.dumps(message),
+                            content_type=JSON, status=200)
+    elif 'error' in message:
+        logging.error(message['error'])
+        return HttpResponse(content=json.dumps(message),
+                            content_type=JSON, status=403)
+    else:
```

Comment:
I don't see how this can happen unless you break the implementation of `share_job`, so I would just add a comment
```
# NB: share_job must return 'success' or 'error'
```
and no else clause. Otherwise, if you want to be really picky
```python
else:
    raise AssertionError("share_job must return 'success' or 'error' !?")
```

Reply:
Done